src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -536,14 +519,35 @@ def get_oidvalue(self, oid):
         return self.oid_map[oid]
 
 class Namespace:
+
+    @staticmethod
+    def init_sonic_db_config():
+        """
+        Initialize SonicDBConfig
+        """
+        # SonicDBConfig has 2 initialize flags: isGlobalInit for multi asic and isInit for single asic
+        if multi_asic.is_multi_asic():
+            if not SonicDBConfig.isGlobalInit():
+                # Load the global config file database_global.json once.
+                SonicDBConfig.load_sonic_global_db_config()
+        elif not SonicDBConfig.isInit():
```

Comment:
>if not SonicDBConfig.isInit(): [](http://example.com/codeflow?start=10&length=30)

No need to check.

Reply:
Fixed, according to another comments, create a new method db_config_loaded()